apibinding/examples/example_py_3_11.py

```
@@ -0,0 +1,87 @@
+import http.client
+import json
+import time
+
+# return a dict containing API return value
+def api_call(session_uuid, api_id, api_content):
+    conn = http.client.HTTPConnection("localhost", 8080)
+    headers = {"Content-Type": "application/json"}
```

Ruff (0.15.4)

[error] Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)